torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -525,6 +525,13 @@ def _get_inverse_affine_matrix(
 
 
 def _compute_affine_output_size(matrix: List[float], w: int, h: int) -> Tuple[int, int]:
+    if torch._dynamo.is_compiling() and not torch.jit.is_scripting():
```

Comment:
Curious why the `and not torch.jit.is_scripting()` part was needed? Is it for cases like `backend="torchscript"`?

Reply:
`torch._dynamo.is_compiling()` does not forbid torchscript to enter to the branch